### PR TITLE
ActiveMentors frontend to backend linking

### PR DESCRIPTION
**Pull request by GalenWong**

### CODE REVIEW

File: client/src/components/ActiveMentors/ActiveMentors.jsx [hunk 1]
```diff
@@ -12,8 +12,8 @@ const iconStyles = {
 class ActiveMentors extends Component {
 	constructor(props){
 		super(props);
+		this.props.getActiveMentors();
```
Comment (shannonphu) [inline client/src/components/ActiveMentors/ActiveMentors.jsx]: Move this call underneath the state initialization

File: client/src/components/ActiveMentors/ActiveMentors.jsx [hunk 2]
```diff
@@ -24,8 +24,8 @@ class ActiveMentors extends Component {
 				<CardText>
 					<p>
 						<FontIcon className='material-icons' style={iconStyles}>lens</FontIcon>
-						<strong>{` `}{this.state.numActive}</strong>
-						{`${this.state.numActive > 1 ?' mentors':' mentor'}
+						<strong>{` `}{this.props.user.mentors.length}</strong>
```
Comment (shannonphu) [inline client/src/components/ActiveMentors/ActiveMentors.jsx]: You could probably store this in a variable in `render()` as to not call `.length` twice.

File: client/src/reducers/user.js [hunk 3]
```diff
@@ -1,10 +1,15 @@
-function UserReducer(state = {}, action) {
+function UserReducer(state = { mentors:[] }, action) {
```
Comment (shannonphu) [inline client/src/reducers/user.js]: Keep spacing consistent. Put a space between the `:` and `[]`

**Comment by TimothyGu** [comment]
This needs to be rebased on master (or merge from master).

File: client/src/components/ActiveMentors/ActiveMentors.jsx [hunk 4]
```diff
@@ -13,19 +13,20 @@ class ActiveMentors extends Component {
...
 	render() {
+		const numActive = this.props.user.mentors.length;
 		return(
 			<Card>
 				<CardText>
 					<p>
 						<FontIcon className='material-icons' style={iconStyles}>lens</FontIcon>
-						<strong>{` `}{this.state.numActive}</strong>
-						{`${this.state.numActive > 1 ?' mentors':' mentor'}
+						<strong>{` `}{numActive}</strong>
```
Comment (shannonphu) [inline client/src/components/ActiveMentors/ActiveMentors.jsx]: You don't need different brackets for this. 
```
{`${this.state.numActive > 1 ?' mentors ':' mentor '}   <---- space here
<strong>{numActive}</strong>
```

**Pull request merged**